deployment/terraform/assets/cluster.tf

```diff
@@ -305,10 +305,12 @@ resource "aws_instance" "loadtest_agent" {
     host = self.public_ip
   }
 
-  ami           = "ami-0ac80df6eff0e70b5"
-  instance_type = var.agent_instance_type
-  key_name      = aws_key_pair.key.id
-  count         = var.agent_instance_count
+  ami                         = "ami-0ac80df6eff0e70b5"
+  instance_type               = var.agent_instance_type
+  key_name                    = aws_key_pair.key.id
+  count                       = var.agent_instance_count
+  subnet_id                   = var.cluster_subnet_id
```

Comment:
@streamer45 Just a question on that. Noticed that we are providing the subnet_id only for the loadtest agent. Don't we need that for the other instances as well?

Reply:
If we wanted to support deploying everything under the same subnet/VPC then yes but my understanding was that you only require the agents, everything else (MM app cluster and Prometheus/Grafana) is deployed separately so here I am just covering this special case. If that's not sufficient we can extend this to all other instances.

Reply:
Sorry didn't get that metrics and app was referring to the Mattermost app and the monitoring part. Makes total sense, thanks for making it clear 👍